src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -771,29 +783,12 @@ public void setShowRawYaml(boolean showRawYaml) {
         this.showRawYaml = Boolean.valueOf(showRawYaml);
     }
 
-    private String getContainersDescriptionForLogging() {
-        List<ContainerTemplate> containers = getContainers();
-        StringBuilder sb = new StringBuilder();
-        for (ContainerTemplate ct : containers) {
-            sb.append("* [").append(ct.getName()).append("] ").append(ct.getImage());
-            StringBuilder optional = new StringBuilder();
-            optionalField(optional, "resourceRequestCpu", ct.getResourceRequestCpu());
-            optionalField(optional, "resourceRequestMemory", ct.getResourceRequestMemory());
-            optionalField(optional, "resourceLimitCpu", ct.getResourceLimitCpu());
-            optionalField(optional, "resourceLimitMemory", ct.getResourceLimitMemory());
-            if (optional.length() > 0) {
-                sb.append("(").append(optional).append(")");
-            }
-            sb.append("\n");
-        }
-        if (isShowRawYaml()) {
-            for (String yaml : getYamls()) {
-                sb.append("yaml:\n")
-                    .append(yaml)
-                    .append("\n");
-            }
+    private String getContainersDescriptionForLogging(KubernetesSlave slave) {
+        if (slave != null && isShowRawYaml()) {
+            return Serialization.asYaml(build(slave));
+        } else {
+            return StringUtils.EMPTY;
```

Comment:
If I read things right, this means when `!showRamYaml`, you get nothing, whereas before you at least got a summary of containers?

Reply:
As it is, we are building the `Pod` multiple times, and it is not clear to me why we would need to do that.

Reply:
Yes, not sure how useful the previous output really was.